packages/react/src/experimental/Await.spec.tsx

```diff
@@ -96,3 +96,30 @@ describe('useAwait', () => {
     await waitFor(() => expect(screen.queryByText(TEXT)).toBeInTheDocument())
   })
 })
+
+describe('Await component', () => {
```

Comment:
I want to change this line like this if we want to describe it as component more intuitively, after changing this file in this Pull Request, Let's change all files like this. Do you agree this? 

```suggestion
describe('<Await/>', () => {
```

Reply:
Certainly, using the JSX style like `<Await />` to represent components can be beneficial as it immediately signifies that it's a React component. However, as you've reviewed with me, [bb1b64a](https://github.com/suspensive/react/pull/169/commits/bb1b64ae81a2d006274c8a4a05e2b48735b3d791) most of the mentions were in the context of describe texts. Shifting to a JSX naming style might deviate from conventional practices. Also, standardization should be taken into consideration. Official documentation, tutorials, and community guidelines of testing libraries or frameworks generally don't recommend or mention using a JSX style for naming. Therefore, this style might differ from standard conventions or practices.

Have you ever seen this(<Await />) convention in the official testing documentation?

Reply:
When I see this [new official docs of react](https://react.dev/reference/react/components), I thought it's pretty intuitive way to express it as component than just using name. regardless of React's new official documentation, I think this way is a pretty good if the everyone can intuitively feel the code. then how about going our own way, even if we have nothing to refer to

![image](https://github.com/suspensive/react/assets/61593290/d5499f43-f375-4750-bb6a-8d4bddd1abe0)

Reply:
I like it. It's definitely clear. I believe there won't be any issues as long as we maintain consistency moving forward. 

Should we reflect this now? Or should listen to Chanhyuk’s opinion?

Reply:
@okinawaa How do you think this?

Reply:
Thank you for your valuable comments and reference At first, the "<Await/>" suggested by manudeli is the most impressive because **I think the test code is easy to read if it is similar to the way the component is actually used.** It's because we're developing it in the "<Await/>" way, not "<Await>"

Reply:
b4bba05